packs/spells/tree-of-seasons.json

```diff
@@ -0,0 +1,72 @@
+{
+    "_id": "u5FAOOXuqK7fLqKW",
+    "img": "icons/environment/wilderness/tree-oak.webp",
+    "name": "Tree of Seasons",
+    "system": {
+        "area": null,
+        "category": {
+            "value": "spell"
+        },
+        "components": {
+            "somatic": true
+        },
+        "cost": {
+            "value": ""
+        },
+        "damage": {
+            "value": {}
+        },
+        "description": {
+            "value": "<p>You cause a Small tree to instantly sprout in an unoccupied space on the ground. Four seedpods grow from the tree, each filled with the magic of a different one of the four seasons. A creature can Interact to pluck one of the pods, and can then either throw it up to 30 feet as part of the same action or do so with a separate Interact action later. When thrown, a pod explodes in a @Template[type:burst|distance:5], dealing 6d6 damage with a basic Reflex save against your spell DC. The damage type depends on the season of the pod: electricity for spring, fire for summer, poison for autumn, or cold for winter. When the spell ends, the tree withers away and any remaining pods rot, leaving behind non-magical seeds.</p>\n<p>@Damage[(@item.level)d6[cold]]</p>\n<p>@Damage[(@item.level)d6[electricity]]</p>\n<p>@Damage[(@item.level)d6[fire]]</p>\n<p>@Damage[(@item.level)d6[poison]]</p>\n<hr />\n<p><strong>Heightened (+1)</strong> The burst's damage increases by 1d6.</p>"
```

Comment:
This might be better handled with overlays?

Reply:
Each casting is all four damage types, one pod of each.

Reply:
Right you are.